src/login/components/App/App_reducer.js

```diff
@@ -1,15 +1,27 @@
 import * as actions from "./App_actions";
+import * as loadingDataActions from "../../redux/actions/loadingDataActions";
 
 const appData = {
-  is_loaded: false
+  is_loaded: false,
+  loading_data: null,
```

Comment:
state.loading_data will update true or false, so I think it would be better if the initial state will be false